common/src/main/java/feast/common/logging/interceptors/GrpcMessageInterceptor.java

```diff
@@ -86,6 +87,7 @@ public void sendMessage(RespT message) {
             entryBuilder.setResponse((Message) message);
           }
 
+          @SneakyThrows
```

Comment:
Please don't use this lombok annotation. We are only using Lombok for getters and setters.